presto-main/src/test/java/com/facebook/presto/operator/scalar/TestQuantileDigestFunctions.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.airlift.stats.QuantileDigest;
+import com.facebook.presto.spi.PrestoException;
+import com.facebook.presto.spi.type.SqlVarbinary;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spi.type.TypeParameter;
+import com.facebook.presto.type.TypeRegistry;
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+import static com.facebook.presto.spi.type.QuantileDigestParametricType.QDIGEST;
+import static io.airlift.slice.Slices.wrappedBuffer;
+import static java.lang.String.format;
+import static java.util.Arrays.asList;
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertEquals;
+
+public class TestQuantileDigestFunctions
+        extends AbstractTestFunctions
+{
+    private static final TypeRegistry TYPE_REGISTRY = new TypeRegistry();
+    private static final Type QDIGEST_BIGINT = QDIGEST.createType(TYPE_REGISTRY, ImmutableList.of(TypeParameter.of(BIGINT)));
+
+    @Test
+    public void testNullQuantileDigestGetValueAtQuantile()
+    {
+        functionAssertions.assertFunction("value_at_quantile(CAST(NULL AS qdigest(bigint)), 0.3)", BIGINT, null);
+    }
+
+    @Test(expectedExceptions = PrestoException.class)
+    public void testGetValueAtQuantileOverOne()
+    {
+        QuantileDigest qdigest = new QuantileDigest(1);
+        functionAssertions.assertFunction(format("value_at_quantile(CAST(X'%s' AS qdigest(bigint)), 1.5)", toHexString(qdigest)),
+                BIGINT,
+                null);
+    }
+
+    @Test(expectedExceptions = PrestoException.class)
```

Comment:
add expected regexp, to avoid future changes change the actual exception being thrown.